libs/application/templates/parental-leave/src/forms/ParentalLeaveForm.ts

```diff
@@ -414,6 +415,31 @@ export const ParentalLeaveForm: Form = buildForm({
                 },
               ],
             }),
+            buildMultiField({
+              id: 'employer.selfEmployed.attachment',
+              title: parentalLeaveFormMessages.selfEmployed.attachmentTitle,
+              description:
+                parentalLeaveFormMessages.selfEmployed.attachmentDescription,
+              condition: (answers) =>
+                (answers as {
+                  employer: {
+                    isSelfEmployed: string
+                  }
+                })?.employer?.isSelfEmployed === YES,
```

Comment:
I was going to suggest that you would:

```typescript
import { SchemaFormValues } from '../lib/dataSchema'
```

And then:
```typescript
{
  // ...
  condition: (answers: SchemaFormValues) => answers.employer?.isSelfEmployed === YES
}
```

But then I noticed `employer` does not exist in the `dataSchema` so you may want to add that as well and then you can try this out.

Reply:
We discussed about it together on Slack. It will need the rework on the schema validators function in order to add it back to the data schema